sui/src/sui_commands.rs

```diff
@@ -468,12 +507,36 @@ async fn make_authority(
     authority: &AuthorityPrivateInfo,
     buffer_size: usize,
     state: AuthorityState,
+    consensus_committee: &ConsensusCommittee<Ed25519PublicKey>,
+    consensus_store_path: &PathBuf,
```

Comment:
It would be more idiomatic to have `&Path`

Reply:
fixing this

Reply:
fixed